src/svc/invocable-scripts/paths/scripts_scriptID.yml

```
@@ -4,33 +4,77 @@ get:
     - Data I/O endpoints
     - Invokable Scripts
   summary: Retrieve a script
-  description: Uses script ID to retrieve details of an invokable script.
+  description: |
+    Retrieves an invokable script.
+
+    #### Related Guides
+
+    - [Invoke custom scripts](https://docs.influxdata.com/influxdb/cloud/api-guide/api-invokable-scripts/)
   parameters:
     - in: path
       name: scriptID
       schema:
         type: string
       required: true
-      description: The script ID.
+      description: |
+        Script ID.
+        Only returns the script with this ID.
```

```suggestion
        The ID of the script to retrieve.
```

I suppose we could apply that same pattern (like you did here) to all the `GET ...${ID}` operations. Makes things easier for us and is more descriptive.
I've following [Square's convention](https://developer.squareup.com/reference/square/subscriptions-api/update-subscription) with "The ID of the...".
What do you think? We should still lead with the article, though: "The script ID."

---

I think i like how square is wording it. "The ID of the..." sounds good to me